style.css

```diff
@@ -40,7 +41,7 @@ body {
   line-height: 60px;
 }
 
-.calculator .orange  {
+.calculator .operator {
```

Comment:
`.operator` needs to be namespaced under `.calculator-` so maybe `.calculator-operator-button` ?